eshiol/plg_content_shorturl
# rel shortlink

###Summary of Changes
Shortlink Link Tag `<link rel="shortlink" href"">`

###Testing Instructions
Enable the option Show Shortlink Link Tag
![shorturl0004](https://user-images.githubusercontent.com/12718836/35747618-2c136992-084b-11e8-8880-d64d4c51611f.png)

###Expected result
The html pahe contains the shortlink link tag
```
<html>
  <head>
    <link rel="shortlink" href"">
```

###Documentation Changes Required
